src/services/ui/src/features/dashboard/index.tsx

```diff
@@ -1,8 +1,10 @@
-import { useEffect, useMemo } from "react";
+import { useMemo } from "react";
 import { QueryClient } from "@tanstack/react-query";
+import { Plus as PlusIcon } from "lucide-react";
```

Comment:
I also added the icon to the button to match the figma since I was there already